tests/test_addons_toupdate.py

```diff
@@ -0,0 +1,610 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
+
+from subprocess import check_call
+
+from click.testing import CliRunner
+
+from acsoo.main import main
+from acsoo.tools import working_directory
+
+
+def _checkout_to(tmpdir, git_ref):
+    check_call(['git', 'checkout', git_ref, '--quiet'], cwd=tmpdir)
+
+
+def test_addon_toupdate_00(initdir):
+    """
+    Data:
+        - addon_1 exists
+        - no requirements file
+    Test case:
+        - compare against the same revision
+        - compare requirements: YES
```

Comment:
s/YES/NO/? because there is no `-r` in the test, or it's rather the `-r` is missing in the test? But then then the exit code should be 0. Strange.